src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsaccreditedprogrammesapi/config/WebClientConfiguration.kt

```diff
@@ -74,7 +74,7 @@ class WebClientConfiguration(
   ): WebClient {
     val oauth2Client = ServletOAuth2AuthorizedClientExchangeFilterFunction(authorizedClientManager)
 
-    oauth2Client.setDefaultClientRegistrationId("prisonerSearchApiWebClient")
+    oauth2Client.setDefaultClientRegistrationId("prisoner-search-api")
```

Comment:
Can we edit the commit before this one to include this change so we're not introducing an incorrect bit of code and then fixing it in the next commit? Should be able to do this with a `git rebase` and then editing the previous commit to include this one.

Reply:
Agreed.